web/src/notifications.js

```diff
@@ -312,6 +302,26 @@ export function process_notification(notification) {
             break;
     }
 
+    return title;
+}
+
+export function process_notification(notification) {
+    const message = notification.message;
+    const content = get_notification_content(message);
+    const key = get_notification_key(message);
+    let notification_object;
+    let msg_count = 1;
+
+    debug_notification_source_value(message);
+
+    if (notice_memory.has(key)) {
+        msg_count = notice_memory.get(key).msg_count + 1;
+        notification_object = notice_memory.get(key).obj;
+        notification_object.close();
+    }
+
+    const title = get_notification_title(message, key, content, msg_count);
```

Comment:
We should not need to pass `key` to this function; I believe the conditional inside `get_notification_title` can just be on whether `msg_count > 1`

Reply:
Yeah, I agree. I guess it should be `msg_count > 0` or `msg_count >= 1`, right?

Reply:
Fixed this.